src/components/lib/patina/patina_mutex.c

```diff
@@ -0,0 +1,124 @@
+/**
+ * Redistribution of this file is permitted under the BSD two clause license.
+ *
+ * Copyright 2020, The George Washington University
+ * Author: Bite Ye, bitye@gwu.edu
+ */
+
+#include <llprint.h>
+#include <patina_mutex.h>
+#include <patina_types.h>
+#include <static_slab.h>
+
+/**
+ * This is a wrapper of patina's native crt lib.
+ * For detail API usage guide, please check source codes
+ * under 'src/components/lib/crt/'
+ */
+
+SS_STATIC_SLAB(lock, struct crt_lock, PATINA_MAX_NUM_MUTEX);
+
+/**
+ * Create a mutex
+ * Currently we don't support any flags
+ *
+ * Arguments:
+ * - @flags: no effect
+ *
+ * @return: id of the mutex
+ */
+patina_mutex_t
+patina_mutex_create(size_t flags)
+{
+	struct crt_lock *l = ss_lock_alloc();
+	if (!l) { return -1; }
```

Comment:
don't need `{}` on same line. I'll avoid repeating this from now on to lighten the notification load.